src/lib/form-field/form-field.ts

```diff
@@ -53,6 +53,7 @@ let nextUniqueId = 0;
   moduleId: module.id,
   // TODO(mmalerba): the input-container selectors and classes are deprecated and will be removed.
   selector: 'mat-input-container, mat-form-field',
+  exportAs: 'matFormField, matInputContainer',
```

Comment:
Since `<mat-input-container>` is being deprecated, should we not include it in `exportAs`

Reply:
Makes sense. Removed it